src/main/java/agh/edu/pl/repackaging/classes/OpenTelemetryClasses.java

```diff
@@ -19,14 +19,12 @@
   private final File mainFile;
   private JarFile agentJar;
   private final String lastFolder;
-  String ADD_OPENTELEMETRY_CLASSES = "./FINAL";
-  String TMP_FOLDER = "./TMP_CLASSES";
 
   public OpenTelemetryClasses(File mainFile, String agentPath, String lastFolder) {
```

Comment:
I find this class name a bit confusing, maybe something like `AgentClassesExtractor` or `AgentClassesCopier` would be more descriptive